### DISCUSSION TITLE
How to add image to repo in search results?

### DISCUSSION THREAD

**Discussion by coffebar** [community/community#47767] [Repositories] [answered]
### Select Topic Area

Question

### Body

When you see search results by topics, some repos have an image, some don't. But i see that image can be missing even if README.md contains it. 
How to fix repo meta to see my screenshot in such list?

![image](https://user-images.githubusercontent.com/3100053/219658246-fa07ce27-dd81-4afa-bb16-db5222043466.png)

**Comment by coffebar** [comment, marked as answer]
Find out. 
Repo settings -> Social Preview -> Upload image

**Reply by sgiraz** [reply]
⚠️ Hi @coffebar It seems does not work. I uploaded the image:
![image](https://user-images.githubusercontent.com/18039045/221381575-a5f2032a-5d01-4770-8db4-386d3c732379.png)

Then I refreshed the browser and the image disappeared:
![image](https://user-images.githubusercontent.com/18039045/221381619-b65ebcaa-5125-4852-bd68-63acb84c68c2.png)
